### DISCUSSION TITLE
Answer on different HTTPS port?

### DISCUSSION THREAD

**Discussion by jfmatth** [piku/piku#320] [Q&A]
Great project!

I'm trying to run PIKU behind my K3s cluster via external service endpoints.

I'd like NGINX to answer SSL on a different port, and I thought the PORT option would do that, but don't see it working?

Am I wrong or just going down a path that doesn't exist?

**Comment by chr15m** [comment]
The port option tells your underlying service to run on a different port and tells nginx to proxy to that port, rather than a random one. There's no way currently to get nginx itself to listen on a different port I don't think.

**Reply by jfmatth** [reply]
thanks.  I think i need to get my ingress to handle it.
